guides/common/modules/proc_configuring-host-based-access-control-for-freeipa-users-logging-in-to-project.adoc

```diff
@@ -0,0 +1,114 @@
+[id="configuring-host-based-access-control-for-freeipa-users-logging-in-to-project_{context}"]
+= Configuring host-based access control for {FreeIPA} users logging in to {Project}
+
+You can use host-based access control (HBAC) rules to manage access control within your {FreeIPA} domain.
+In {FreeIPA}, HBAC rules define which users can access which hosts and which services can be used to gain access.
+
+For example, you can configure HBAC on the {FreeIPA} server to limit access to {ProjectServer} only to selected users or user groups.
+By configuring a HBAC rule in the {FreeIPA} domain, you can ensure {Project} does not create database entries for users who should not have access.
+
+.Prerequisites
+* {FreeIPA} user account with privileges to configure HBAC rules
```

Comment:
```suggestion
* {FreeIPA} user account with privileges to configure HBAC rules.
```

unsure. Maybe start with "You have a ..."?

Reply:
I think "You have" kind of goes without saying..? I don't see much value in adding those two words.

And only list items in lists where at least one item is a complete sentence should end with a period (IBM style guide -- if the convention in Foreman docs is different, but so far, I've been applying the IBM style guide for lists).

No change.